CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   - `whole_file_hash`: By default only the first chunk of the uploaded import is hashed to save time. When this is set to True each uploaded file is fully hashed instead.
 - Added parameters `external_access_integrations` and `secrets` when creating a UDAF from Snowpark Python to allow integration with external access.
 - `SessionBuilder.getOrCreate` will now attempt to replace the singleton it returns when token expiration has been detected.
+- Added support for `Interval` expressions.
```

Comment:
Since this implementation does not add a public API, I don't think we should call this out in the changelog. We should decide upon what is the right way to expose this in the public api.

Reply:
Removed from changelog. For figuring out how to expose this (and other snowpark functionality used by snowpark pandas) as a public API, filed this JIRA ticket https://snowflakecomputing.atlassian.net/browse/SNOW-1002101.